scipy/stats/tests/test_contingency.py

```diff
@@ -2,9 +2,9 @@
 from numpy.testing import (assert_equal, assert_array_equal,
          assert_array_almost_equal, assert_approx_equal, assert_allclose)
 from pytest import raises as assert_raises
-
 from scipy.special import xlogy
-from scipy.stats.contingency import margins, expected_freq, chi2_contingency
+from scipy.stats.contingency import margins, expected_freq, \
+    chi2_contingency, association
```

Comment:
We generally don't use the continuation character when there is an alternative, such as
```suggestion
from scipy.stats.contingency import (margins, expected_freq,
                                     chi2_contingency, association)
```